lvk/vulkan/VulkanClasses.cpp

```diff
@@ -2955,8 +2962,10 @@ lvk::VulkanStagingDevice::VulkanStagingDevice(VulkanContext& ctx) : ctx_(ctx) {
 
   const VkPhysicalDeviceLimits& limits = ctx_.getVkPhysicalDeviceProperties().limits;
 
+  maxStorageBufferSize_ = limits.maxStorageBufferRange;
+
   // use default value of 128Mb clamped to the max limits
-  maxBufferSize_ = std::min(limits.maxStorageBufferRange, 128u * 1024u * 1024u);
+  maxBufferSize_ = std::min(maxStorageBufferSize_, 128u * 1024u * 1024u);
```

Comment:
Why do we need an extra member field? We can use `limits.maxStorageBufferRange` directly.